### PR TITLE
KFSPTS-19100 add inactive date to position data

### PR DESCRIPTION
**Pull request by db483**

### CODE REVIEW

**Comment by db483** [comment]
Please review code changes but do not merge yet. I am waiting for Janet to confirm the label and display of new field. Thank you.

**Review by jhulslander — COMMENTED** [review]
This change looks good, I just had one question

File: src/main/java/edu/cornell/kfs/module/ld/businessobject/PositionDataExtentedAttribute.java [hunk 1]
```diff
@@ -1,49 +1,51 @@
...
-		 
-	  }
+    private static final long serialVersionUID = 1L;
+
+    private String positionNumber;
+    private String orgCode;
+    private Date effectiveDate;
+    private Date inactivationDate;
+
+    public PositionDataExtentedAttribute() {
```
[the remark this answers is not in the document]
Comment (db483) [reply src/main/java/edu/cornell/kfs/module/ld/businessobject/PositionDataExtentedAttribute.java:15]: I am not sure that it is needed. It was there before, I just updated the formatting.

[the remark this answers is not in the document]
Comment (jhulslander) [reply src/main/java/edu/cornell/kfs/module/ld/businessobject/PositionDataExtentedAttribute.java:15]: OK, in the interest of moving this along, lets keep it

**Review by jhulslander — APPROVED** [review]
my concerns have been discussed, the failed unit tests are related to utest fresh and will be fixed on another user story.  I shall merge

**Pull request merged**